pronto/api/signature.py

```diff
@@ -43,14 +43,17 @@ def get_signature(accession):
     row = cur.fetchone()
     cur.close()
     con.close()
-
+    
     if not row:
         return jsonify({
             "error": {
                 "title": "Signature not found",
-                "message": "Entry names cannot be longer than 100 characters."
+                "message": f"<strong>{accession}</strong> does not match any member database signature accession or name."
```

Comment:
Remove the HTML from the message:

```suggestion
                "message": f"<{accession} does not match any member database signature accession or name."
```